modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -24,6 +24,7 @@ $ sudo dnf install -y podman
 [source,terminal]
 ----
 $ sudo firewall-cmd --add-port=8080/tcp --zone=public --permanent
+$ sudo firewall-cmd --reload
```

Comment:
There shouldn't be more than one command in a code block (per our guidelines here: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output)

Do you mind updating to put this command in a separate code block?

Reply:
Added `sudo firewall-cmd --reload` as a new code block.